apps/console/src/features/claims/pages/claim-dialects.tsx

```diff
@@ -166,15 +167,18 @@ const ClaimDialectsPage: FunctionComponent<ClaimDialectsPageInterface> = (
             ) }
             <PageLayout
                 action={
-                    attributeConfig.addAttributeMapping && <PrimaryButton
-                        onClick={ () => {
-                            setAddEditClaim(true);
-                        } }
-                        data-testid={ `${ testId }-list-layout-add-button` }
-                    >
-                        <Icon name="add" />
-                        { t("console:manage.features.claims.dialects.pageLayout.list.primaryAction") }
-                    </PrimaryButton>
+                    attributeConfig.addAttributeMapping &&
```

Comment:
Shall we wrap with `()`? WDYT?
```suggestion
                    attributeConfig.addAttributeMapping && (
                        ...
                    ) }    
```